spyderlib/plugins/shortcuts.py

```diff
@@ -15,30 +15,29 @@
                                 QPushButton, QKeySequence, QDialog,
                                 QDialogButtonBox, QLabel, QGridLayout,
                                 QLineEdit, QAbstractItemView,
-                                QSortFilterProxyModel, QStyledItemDelegate,
-                                QStyleOptionViewItem, QApplication,
-                                QTextDocument, QStyle, QSpacerItem,
-                                QAbstractTextDocumentLayout,
-                                QRegExpValidator, QToolTip,
-                                QHBoxLayout)
-from spyderlib.qt.QtCore import (Qt, QAbstractTableModel, QModelIndex, QSize,
-                                 QPoint, QRegExp)
+                                QSortFilterProxyModel, QApplication,
+                                QSpacerItem, QRegExpValidator, QHBoxLayout)
+from spyderlib.qt.QtCore import Qt, QAbstractTableModel, QModelIndex, QRegExp
 from spyderlib.qt.compat import to_qvariant, from_qvariant
-import spyderlib.utils.icon_manager as ima
 
 # Local imports
 from spyderlib.baseconfig import _, debug_print
 from spyderlib.guiconfig import (get_shortcut, set_shortcut,
                                  iter_shortcuts, reset_shortcuts)
 from spyderlib.plugins.configdialog import GeneralConfigPage
+from spyderlib.utils import icon_manager as ima
 from spyderlib.utils.qthelpers import get_std_icon
+from spyderlib.utils.stringmatching import get_search_regex, get_search_scores
 from spyderlib.widgets.arraybuilder import HelperToolButton

```

Comment:
This helper button should be moved to `helperwidgets` too. But this is just a reminder, let's leave it for another PR :-)